scripts/cddl_lexer.py

```diff
@@ -0,0 +1,76 @@
+import re
+
+from pygments.lexer import RegexLexer, bygroups, words
```

Comment:
Are these imports provided by the pygments modules that Bikeshed requires?  If so, we won't need to ask contributors to install other stuff to continue to build our spec, which is a win.

Reply:
Yeah, you don't have to do anything fancy to get custom pygment lexers working. Getting bikeshed set up should be enough.